modules/persistent-storage-csi-google-cloud-file-create-sc.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-google-cloud-file.adoc
+
+:_content-type: PROCEDURE
+[id="persistent-storage-csi-google-cloud-file-create-sc_{context}"]
+= Creating a storage class for GCP Filestore Storage
+
+After installing the operator, you should create a storage class for dynamic provisioning of Google Compute Platform (GCP) Filestore volumes. 
+
+.Prerequisites
+* You are logged in to the running {product-title} cluster.
+
+.Procedure
+To create a storage class:
+
+. Create a storage class using the following example YAML file:
++
+[source,yaml]
+.Example YAML file
+--
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: filestore-csi
+provisioner: filestore.csi.storage.gke.io
+parameters:
+  network: network-name <1>
+allowVolumeExpansion: true
+volumeBindingMode: WaitForFirstConsumer
+--
+<1> Specify the name of the network where Filestore instances should be created in. 
```

Comment:
`network` -> `GCP VPC Network` or `GCP Virtual Private Cloud (VPC) Network`.

Reply:
This is the Google docs on VPC, for reference: https://cloud.google.com/vpc/docs/vpc

Reply:
Done.